k8s/gitlab.cronjob.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eclipsefdn-github-sync
```

Comment:
I think this should be changed to eclipsefnd-gitlab-sync or else it will clobber the existing 'cronjob'